openml/_api_calls.py

```diff
@@ -456,6 +459,8 @@ def __parse_server_exception(
     url: str,
     file_elements: FILE_ELEMENTS_TYPE | None,
 ) -> OpenMLServerError:
+    # You can find the individual codes that are parsed out of the response here:
+    # https://github.com/openml/OpenML/blob/develop/openml_OS/views/pages/api_new/v1/xml/pre.php
```

Comment:
This would be better a few lines down with "OpenML has a sophisticated error system".

This magic number should probably be replaced by `HTTPStatus.URI_TOO_LONG` since it's a HTTP header code, not an openml body code.